packages/neuron-wallet/src/services/addresses.ts

```diff
@@ -5,9 +5,15 @@ import LockUtils from 'models/lock-utils'
 import AddressDao, { Address as AddressInterface } from 'database/address/dao'
 import env from 'env'
 import AddressEntity, { AddressVersion } from 'database/address/entities/address'
+import AddressCreatedSubject from 'models/subjects/address-created-subject'
 
 const MAX_ADDRESS_COUNT = 30
 
+export interface AddressWithWay {
```

Comment:
The name is confusing, how about `AddressAndIsImporting`

Reply:
`isImporting` moves to `Address` structure, so this struct deleted